src/udfs/pytorch_abstract_udf.py

```diff
@@ -43,21 +44,25 @@ def get_device(self):
     def transforms(self) -> Compose:
         return Compose([transforms.ToTensor()])
 
-    def transform(self, images: Any):
-        return self.transforms(images).to(self.get_device())
+    def transform(self, images: np.ndarray):
+        # reverse the channels from opencv
+        return self.transforms(Image.fromarray(images[:, :, ::-1]))\
+            .unsqueeze(0)
 
-    def forward(self, frames: np.ndarray):
-        return self.classify([self.transform(frame[0]) for frame in frames])
+    def forward(self, frames: List[np.ndarray]):
+        tens_batch = torch.cat([self.transform(x) for x in frames])\
+            .to(self.get_device())
```

Comment:
Is this done to speed up the data transfer?

Reply:
This is batched (multiple frames) model for pytorch. But From the latency numbers I get.  I do not see performance improvement.